preparing-to-code/keyboard-navigation/README.md

```diff
@@ -1,55 +1,123 @@
+
+| Button    | Mac Symbol | How To Remember It                                                                                      |
+| --------- | :--------: | ------------------------------------------------------------------------------------------------------- |
+| Command   |     ⌘      | Ah, here's where I left those four robots I can command to open menus / do other computer stuff for me. |
+| Option    |     ⌥      | Sliding down a bookcase lined with options & picking the first one that looks good.                     |
+| Control   |     ⌃      | The magic button I need to press to gain control of a robot.                                            |
+| Shift     |     ⇧      | Hey, you short lowercase letter, come up here!                                                          |
+| Caps Lock |     ⇪      | And stay up! Here's a box to help. You can stand on it. :)                                              |
+
+
+
+## Common Hotkeys that will save your life
+
+### Text Editing Keyboard Combinations
+
+Below are a series of keyboard shortcuts that will help you in a text-editor writing code.
+
+| Hot Key Combo              |           Mac Symbols           | What It Does                                                 |
+| -------------------------- | :-----------------------------: | ------------------------------------------------------------ |
+| Command-s                  |             `⌘ + s`             | Save code in the text editor!  **Do this a lot!**            |
+| Command-c                  |             `⌘ + c`             | Copy the text currently highlighted (ctrl-c in windows).     |
+| Command-x                  |             `⌘ + x`             | Remove the text highlighted and save it in the clipboard.    |
+| Command-v                  |             `⌘ + v`             | Paste last copied content from clipboard.                    |
+| Command-z                  |             `⌘ + z`             | Undo the last action (repeat to undo multiple things).       |
+| Command-shift-z            |           `⌘ + ⇧ + z`           | Redo the last undo.                                          |
+| Command-a                  |             `⌘ + a`             | Select everything in the window.                             |
+| Command-l                  |             `⌘ + l`             | Select the current line.                                     |
+| Command-Arrow              |    `⌘ + <directional arrow>`    | Move the cursor to the maximum point in the given direction. |
+| Option-left-or-right-arrow |   `⌥ + <left or right arrow`    | Move the cursor one word in either direction                 |
+| Shift-Arrow                | `⇧ + <cursor movement command>` | Highlight from the start and end of the cursor               |
```

Comment:
It's a clearer if we use the `<kbd>` tag to indicate buttons.  Also, we wrote "ctrl" and "cmd" in other parts of JumpStart so probably want to normalize to these longer ones.

```suggestion
| Hot Key Combo                                        |           Mac Symbols           | What It Does                                                 |
| ---------------------------------------------------- | :-----------------------------: | ------------------------------------------------------------ |
| <kbd>Command</kbd> + <kbd>s</kbd>                    |             `⌘ + s`             | Save code in the text editor!  **Do this a lot!**            |
| <kbd>Command</kbd> + <kbd>c</kbd>                    |             `⌘ + c`             | Copy the text currently highlighted (ctrl-c in windows).     |
| <kbd>Command</kbd> + <kbd>x</kbd>                    |             `⌘ + x`             | Remove the text highlighted and save it in the clipboard.    |
| <kbd>Command</kbd> + <kbd>v</kbd>                    |             `⌘ + v`             | Paste last copied content from clipboard.                    |
| <kbd>Command</kbd> + <kbd>z</kbd>                    |             `⌘ + z`             | Undo the last action (repeat to undo multiple things).       |
| <kbd>Command</kbd> + <kbd>shift</kbd> + <kbd>z</kbd> |           `⌘ + ⇧ + z`           | Redo the last undo.                                          |
| <kbd>Command</kbd> + <kbd>a</kbd>                    |             `⌘ + a`             | Select everything in the window.                             |
| <kbd>Command<kbd> + <kbd>l</kbd>                     |             `⌘ + l`             | Select the current line.                                     |
| <kbd>Command</kbd> + any arrow key                   |    `⌘ + <any arrow>`         | Move the cursor to the maximum point in the given direction. |
| <kbd>Option</kbd> left or right arrow key            |   `⌥ + <left or right arrow>`    | Move the cursor one word in either direction                 |
| <kbd>Shift</kbd> + any arrow key                     | `⇧ + <any arrow>`            | Highlight from the start and  end of the cursor              |
```